src/encoded/submission_status.py

```diff
@@ -122,6 +146,20 @@ def add_submission_status_search_filters(
         search_params["libraries.assay.display_title"] = filter["assay"]
     if "sequencer" in filter and filter["sequencer"] != "all":
         search_params["sequencing.sequencer.display_title"] = filter["sequencer"]
+    if "cell_line" in filter and filter["cell_line"] != "all":
+        search_params["libraries.analytes.samples.sample_sources.cell_line.code"] = (
+            filter["cell_line"]
+        )
+        search_params["libraries.analytes.samples.sample_sources.code"] = (
+            "No+value"  # Exlude mixtures from results
+        )
```

Comment:
I think subsequent call to `urlencode` makes use of `+` unnecessary here, does this work? Also typo: "exclude"